Gordon360/Web.config

```diff
@@ -108,5 +106,6 @@
   </entityFramework>
   <connectionStrings>
     <add name="CCTEntities1" connectionString="metadata=res://*/Models.CCT_DB_Models.csdl|res://*/Models.CCT_DB_Models.ssdl|res://*/Models.CCT_DB_Models.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=admintrainsql.gordon.edu;initial catalog=CCT;integrated security=True;multipleactiveresultsets=True;application name=EntityFramework&quot;" providerName="System.Data.EntityClient" />
+    <add name="CCTEntities" connectionString="metadata=res://*/Models.NewFolder1.CCT_DB_MODELS.csdl|res://*/Models.NewFolder1.CCT_DB_MODELS.ssdl|res://*/Models.NewFolder1.CCT_DB_MODELS.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=admintrainsql.gordon.edu;initial catalog=CCT;integrated security=True;multipleactiveresultsets=True;application name=EntityFramework&quot;" providerName="System.Data.EntityClient" />
```

Comment:
This line seems to be an almost duplicate of the one above. Should we fix that?

Reply:
Change made